packages/nimble-components/src/table/types.ts

```diff
@@ -31,7 +31,11 @@ export type TableStringField<FieldName extends TableFieldName> = {
     [name in FieldName]: TableStringFieldValue;
 };
 
-export interface TableValidity {
+export interface ValidityObject {
+    [key: string]: boolean;
+}
+
+export interface TableValidity extends ValidityObject {
```

Comment:
The Blazor version of this interface needs to get updated in `TableValidity.cs`.